packages/gatsby/src/utils/webpack/plugins/static-query-mapper.ts

```diff
@@ -83,293 +83,293 @@ export class StaticQueryMapper {
-            component.isSlice ? {} : cloneDeep(globalSliceUsage)
-          )
-
-          const slices = Object.keys(allSlices)
-            .sort()
-            .reduce((obj, key) => {
-              obj[key] = allSlices[key]
-              return obj
-            }, {})
-
-          const didSlicesChange = !isEqual(
-            this.store.getState().slicesByTemplate.get(component.componentPath),
-            slices
-          )
-          const didStaticQueriesChange = !isEqual(
-            this.store
-              .getState()
-              .staticQueriesByTemplate.get(component.componentPath),
-            staticQueryHashes
-          )
-
-          if (didStaticQueriesChange || didSlicesChange) {
-            if (component.isSlice) {
-              this.store.dispatch({
-                type: `ADD_PENDING_SLICE_TEMPLATE_DATA_WRITE`,
-                payload: {
-                  componentPath: component.componentPath,
-                  sliceNames: component.pages,
-                },
-              })
-            } else {
-              this.store.dispatch({
-                type: `ADD_PENDING_TEMPLATE_DATA_WRITE`,
-                payload: {
-                  componentPath: component.componentPath,
-                  pages: component.pages,
-                },
-              })
-            }
-          }
-
-          if (didSlicesChange) {
-            this.store.dispatch({
-              type: `SET_SLICES_BY_TEMPLATE`,
-              payload: {
-                componentPath: component.componentPath,
-                slices,
-              },
-            })
-          }
-
-          if (didStaticQueriesChange) {
-            this.store.dispatch({
-              type: `SET_STATIC_QUERIES_BY_TEMPLATE`,
-              payload: {
-                componentPath: component.componentPath,
-                staticQueryHashes,
-              },
-            })
-          }
-        }
-      }
-    )
+    // compiler.hooks.finishMake.tapPromise(
+    //   {
+    //     name: this.name,
+    //     before: `PartialHydrationPlugin`,
+    //   },
+    //   async compilation => {
+    //     if (compilation.compiler.parentCompilation) {
+    //       return
+    //     }
+
+    //     const entryModules = new Set()
+    //     const gatsbyBrowserPlugins = slash(
+    //       path.join(
+    //         program.directory,
+    //         `.cache`,
+    //         `api-runner-browser-plugins.js`
+    //       )
+    //     )
+    //     const asyncRequiresPath = slash(
+    //       path.join(
+    //         program.directory,
+    //         `.cache`,
+    //         `_this_is_virtual_fs_path_`,
+    //         `$virtual`,
+    //         `async-requires.js`
+    //       )
+    //     )
+    //     for (const entry of compilation.entries.values()) {
+    //       for (const dependency of entry.dependencies) {
+    //         const mod = compilation.moduleGraph.getModule(dependency)
```

Comment:
`compilation.moduleGraph` not available in JavaScript. It is used internally in Rust land, but not exposed to JS